3p/ampcontext.js

```diff
@@ -62,24 +120,33 @@ export class AmpContext {
    *    every time we receive a page visibility message.
    */
   observePageVisibility(callback) {
```

Comment:
discussed offline, we'll rename it to `onPageVisibilityChange(callback)` which documents the current behavior clearer (only fires on change).